programs/programs/co/denver_sidewalk_rebate/calculator.py

```diff
@@ -0,0 +1,53 @@
+from integrations.services.sheets.sheets import GoogleSheetsCache
+from programs.co_county_zips import counties_from_screen
+from programs.programs.calc import Eligibility, ProgramCalculator
+import programs.programs.messages as messages
+
+
+class IncomeLimitsCache(GoogleSheetsCache):
+    sheet_id = "1dahRu8UVdWBBU1jMiGiWehY4kOUkzcOmKRPJ-GfcfGo"
+    range_name = "B2:I"
+    default = {}
+
+    def update(self):
+        data = super().update()
+
+        return self._format_amounts(data[0])
+
+    @staticmethod
+    def _format_amounts(amounts: list[str]):
+        return [float(a.strip().replace(",", "")) for a in amounts]
+
+
+class DenverSidewalkRebate(ProgramCalculator):
+    county = "Denver County"
+    income_limits = IncomeLimitsCache()
+    presumptive_eligibility = ["co_medicaid", "snap", "tanf", "cccap"]
+    amount = 150
+    dependencies = ["household_size", "income_amount", "income_frequency"]
```

Comment:
Because you are using county, you need to add `zipcode` to the `dependencies`
```suggestion
    dependencies = ["household_size", "income_amount", "income_frequency", "zipcode"]
```